test-requirements.txt

```diff
@@ -140,7 +140,7 @@ requests-oauthlib==1.3.1
     # via kubernetes
 rsa==4.9
     # via google-auth
-ruff==0.3.1
+ruff==0.3.2
```

Comment:
Blocking merge. I need to use this PR to test the automation introduced by #76 when it is merged.